src/dotnet/Interop.cs

```diff
@@ -979,6 +980,15 @@ internal static Int32 SetDebugLogFile(String FileName)
             return 0/*STATUS_SUCCESS*/;
         }
 
+        internal static Version GetFspVersion()
+        {
+            UInt32 Version = 0, VersionMajor, VersionMinor;
+            FspVersion(out Version);
+            VersionMajor = Version >> 16;
+            VersionMinor = Version & 0xFFFF;
+            return new System.Version((Int32)VersionMajor, (Int32)VersionMinor);
+        }
+
```

Comment:
Looks good overall. I do find the `VersionMajor` and `VersionMinor` variables unnecessary. Perhaps we should change this to read:

```C#
    internal static Version GetFspVersion()
    {
        UInt32 Version = 0;
        FspVersion(out Version);
        return new System.Version((Int32)(Version >> 16), (Int32)(Version & 0xFFFF));
    }
```